plugins/modules/system/sefcontext.py

```diff
@@ -22,6 +22,13 @@
     type: str
     required: yes
     aliases: [ path ]
+  equiv:
+    description:
+    - set target selinux equivalence to this path.
+    - setype must be set, the bogus 'many_t' works.
+    - when set, ftype, seuser, selevel are ignored.
+    type: str
```

Comment:
The next minor release will be 5.6.0:
```suggestion
    version_added: 5.6.0
    type: str
```